spring-cloud-starter-dataflow-server-kubernetes/pom.xml

```diff
@@ -21,5 +21,51 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-server-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Or do this - https://github.com/spring-cloud/spring-cloud-dataflow/issues/1008

That way we don't have to have a banner in every server impl